scripts/s3/s3api/copy_object.py

```diff
@@ -72,23 +70,31 @@ async def execute_copy_object_workload(self):
         object1 = f"object-1-{self.test_id}-{time.perf_counter_ns()}".lower()
         object2 = f"object-2-{self.test_id}-{time.perf_counter_ns()}".lower()
         await self.create_bucket(bucket1)
-        logger.info("Created bucket %s", bucket1)
+        LOGGER.info("Created bucket %s", bucket1)
         await self.create_bucket(bucket2)
-        logger.info("Created bucket %s", bucket2)
+        LOGGER.info("Created bucket %s", bucket2)
         while True:
-            logger.info("Iteration %s is started...", self.iteration)
+            LOGGER.info("Iteration %s is started...", self.iteration)
             try:
                 # Put object in bucket1
-                if not isinstance(self.object_size, dict):
-                    file_size = self.object_size
-                else:
-                    file_size = random.randrange(self.object_size["start"], self.object_size["end"])
-                with open(object1, 'wb') as fout:
-                    fout.write(os.urandom(file_size))
+                file_size = random.randrange(
+                    self.object_size["start"],
+                    self.object_size["end"]) if isinstance(
+                    self.object_size,
+                    dict) else self.object_size
```

Comment:
Same comment as above. Previous code was clean and reasonable to read and understand.

Reply:
Addressed this in previous comment.